src/hpccplatform/launchConfig.ts

```diff
@@ -71,13 +71,12 @@ export function launchConfigurations(refresh = false): string[] {
             gatherServers();
         }
     }
-    const retVal = Object.keys(g_launchConfigurations);
+    const retVal = Object.values(g_launchConfigurations);
     if (retVal.length === 0) {
```

Comment:
@GordonSmith, I'm not sure it can ever get here. If there are no launch configurations, you block it already. Maybe there is a cases where this would get here if zero. Doesn't hurt I guess.

Reply:
I think it can still have 0 entries at this point...